### ISSUE TITLE
configuring requires Module::Build

### ISSUE THREAD

**Issue by skaji** [Ovid/Test-Differences#13]
* Test-Differences has both Build.PL and Makefile.PL.
* Test-Differences requires only ExtUtils::MakeMaker, not Module::Build at configure phase.  
https://metacpan.org/source/DCANTRELL/Test-Differences-0.64/META.json#L28-32
* Some cpan clients (such as cpanminus) try to configure distributions by Build.PL first.

As a result, some cpan clients fail to configure Test-Differences at first trial.
```
> perl -v
This is perl 5, version 24, subversion 0 (v5.24.0) built for darwin-2level
(with 1 registered patch, see perl -V for more detail)

> cpanm -nq -Llocal Test::Differences
...
Successfully installed Test-Differences-0.64

> cat ~/.cpanm/build.log
...
Configuring Test-Differences-0.64
Running Build.PL
Can't locate Module/Build.pm in @INC (you may need to install the Module::Build module) (@INC contains: /Users/skaji/work/2016-12-25/local/lib/perl5 /Users/skaji/env/plenv/versions/t5.24/lib/site_perl/5.24.0/darwin-2level /Users/skaji/env/plenv/versions/t5.24/lib/site_perl/5.24.0 /Users/skaji/env/plenv/versions/t5.24/lib/5.24.0/darwin-2level /Users/skaji/env/plenv/versions/t5.24/lib/5.24.0 .) at Build.PL line 3.
BEGIN failed--compilation aborted at Build.PL line 3.
Running Makefile.PL
Warning: prerequisite Capture::Tiny 0.24 not found.
Warning: prerequisite Text::Diff 1.43 not found.
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Test::Differences
Writing MYMETA.yml and MYMETA.json
```

So could you:
* list Module::Build as configure requires?
* or, remove Build.PL (or Makefile.PL, whichever you want)?

Thank you.

**Comment by DrHyde** [comment]
Thanks, fixed in 3b87de7 and will be on its way to the CPAN soon.

**Comment by skaji** [comment]
@DrHyde Thanks!
